### ISSUE TITLE
Review/ fix stand.age issue

### ISSUE THREAD

**Issue by teixeirak** [forc-db/ForC#55]
from a reviewer:  There are many oddities among the records included with this age designation, including monospecific plantations (e.g., 15 records from La Selva) and many logged S. American forests (diverse sites).

**Comment by teixeirak** [comment]
We replaced the ‘999’ code with a no data code (“NAC”) for the 15 records from La Selva, and we also recorded in the vegetation notes that these are monospecific plantations. Furthermore, we reviewed the site and plot names for all records with the ‘999’ code and replaced with a no data code (“NAC”) for any records that were identifiable as disturbed based on these names (which we selected to be informative). This included some logged S. American forests, but lacking the reviewer’s personal knowledge of neotropical research sites, we cannot be certain that we got them all. This issue may have arisen from the fact that we did not count a partial disturbance as resetting stand age to zero. We have modified the stand.age description to include the following warning: “WARNING: The "999" designation may include some older secondary stands that were designated "mature" or some partially disturbed primary stands (e.g., partially logged).”
